libraries/scripting/type-providers/src/kotlin/script/experimental/typeProviders/generatedCode/impl/makeMethod.kt

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package kotlin.script.experimental.typeProviders.generatedCode.impl
+
+import kotlin.reflect.*
+import kotlin.script.experimental.typeProviders.generatedCode.GeneratedCode
+import kotlin.script.experimental.typeProviders.generatedCode.GeneratedTypeBuilder
+import kotlin.script.experimental.typeProviders.generatedCode.internal.visitor.useImport
+import kotlin.script.experimental.typeProviders.generatedCode.internal.visitor.useImports
+import kotlin.script.experimental.typeProviders.generatedCode.internal.visitor.GeneratedCodeVisitor
+import kotlin.script.experimental.typeProviders.generatedCode.internal.InternalGeneratedCode
+import kotlin.script.experimental.typeProviders.generatedCode.override
+
+private val storedValues = mutableMapOf<String, Any>()
```

Comment:
this is an unacceptable global state. You need to put it into some context carried with the script.

Reply:
Replaced global state with serialized values file